libs/features/personalization/personalization.js

```diff
@@ -239,11 +239,89 @@ function normalizeKeys(obj) {
   }, {});
 }
 
+const getDivInTargetedCell = (tableCell) => {
+  tableCell.replaceChildren();
+  const div = document.createElement('div');
+  tableCell.appendChild(div);
+  return div;
+};
+
+const querySelector = (el, selector, all = false) => {
+  try {
+    return all ? el.querySelectorAll(selector) : el.querySelector(selector);
+  } catch (e) {
+    /* eslint-disable-next-line no-console */
+    console.log('Invalid selector: ', selector);
```

Comment:
Should this be left here for the final version? I see there's also a unit test for this, so I assume so. Should we at least consider logging only on non-prod environments? Is `log` the right choice, or would `warn` be more visible for authors?